app/database/group_groups.go

```diff
@@ -0,0 +1,21 @@
+package database
+
+// GroupRelationIsActiveCondition is an SQL condition restricting a value
+// to be one of ('direct', 'invitationAccepted', 'requestAccepted', 'joinedByCode')
```

Comment:
Instead of writing the values it matches (that the reader can read at the next line), I think we should mention what it means (so that other dev know if they are supposed to use it or not).
So, something like `GroupRelationIsActiveCondition is an SQL condition restricting group types to those matching a current actual membership of a user or group within a group.`
And for the next one:  `GroupUserRelationIsActiveCondition is an SQL condition restricting group types to those matching a current actual user membership within a group.`

Reply:
fixed